### PR TITLE
New wir.by page

### PR DESCRIPTION
**Pull request by UladBohdan**
* styles for landing wir.by are completely rewritten and are independent from legacy code
* bulma is only used as separate modules
* images are moved to cloudinary
* gulpfile exit warnings fixed
* code is separated between `landing` (to be saved), `stubs` and `src` (to be removed)

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -18,7 +18,14 @@ const fs = require('fs');
 const templateVariables = JSON.parse(
   fs.readFileSync('./src/templateVariables.json', 'utf8')
 );
-const team = JSON.parse(fs.readFileSync('./src/about/team.json', 'utf8'));
+
+const teammateImageScaleOptions = 'c_scale,w_240';
+
+let rawTeam = JSON.parse(fs.readFileSync('./src/about/team.json', 'utf8'));
```
Comment (Drapegnik) [inline gulpfile.js]: why not `const`?

File: gulpfile.js [hunk 2]
```diff
@@ -64,6 +72,25 @@ gulp.task('sass:bundle', () =>
     )
 );
 
+gulp.task('sass:bundle', () =>
```
Comment (Drapegnik) [inline gulpfile.js]: duplicate `sass:bundle` task definition

Comment (UladBohdan) [reply gulpfile.js]: is that a problem?

Comment (UladBohdan) [reply gulpfile.js]: Fixed.

File: gulpfile.js (hunk 1 above)
Comment (UladBohdan) [reply gulpfile.js]: Fixed.

**Comment by UladBohdan** [comment]
[to fix Ctrl+C on markup watch process](https://trello.com/c/zOUN5lRx/311-to-fix-ctrlc-on-markup-watch-process)

**Comment by UladBohdan** [comment]
@Drapegnik @andemerie I want these changes to go live asap

File: landing/landing.scss [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+    padding: 1rem;
+    width: 100%;
+
+    @include desktop {
+      margin: 1.5rem 0;
+      width: 18rem;
+    }
+  }
+
+  &__team-member {
```
Comment (Drapegnik) [inline landing/landing.scss]: what's about to extract teammate styles to `teammate.scss`?

Comment (UladBohdan) [reply landing/landing.scss]: I'll think about it

**Pull request merged**